spec/javascript-spec.coffee

```diff
@@ -85,6 +85,13 @@ describe "Javascript grammar", ->
       {tokens} = grammar.tokenizeLine('0O1411')
       expect(tokens[0]).toEqual value: '0O1411', scopes: ['source.js', 'constant.numeric.js']
 
+    it "regular expressions have explicit count modifiers", ->
```

Comment:
That makes sense. I couldn't understand what `it` referred to.
Thanks @kevinsawicki!